src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const state = {
+  temperature: 79,
+  city: 'London',
+  isF: true,
+};
+
+const convertKtoC = (temp) => temp - 273.15;
+const convertKtoF = (temp) => (temp - 273.15) * (9 / 5) + 32;
+const convertCtoF = (temp) => temp * (9 / 5) + 32;
+const convertFtoC = (temp) => (temp - 32) * (5 / 9);
+
+const getLatAndLon = (city) => {
+  return axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const getWeatherKelvin = (lat, lon) => {
+  return axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: lat,
+        lon: lon,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return response.data.current.temp;
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const handleCityEntered = (event) => {
+  console.log(`City entered ${event}`);
+  const inputCityName = event.target.value;
+  const displayCityName = document.getElementById('cityDisplay');
+  state.city = inputCityName;
+  displayCityName.textContent = state.city;
+};
+
+const increaseTemp = (event) => {
+  state.temperature += 1;
+  colorChange();
+};
+
+const decreaseTemp = (event) => {
+  state.temperature -= 1;
+  colorChange();
+};
+
+const reset = (event) => {
+  // state.temperature = 79; //change it after API
+  state.city = 'London';
+  document.getElementById('city').value = state.city;
+  document.getElementById('cityDisplay').textContent = state.city;
+
+  go(event);
+};
+
+const go = (event) => {
+  event.preventDefault()
+  console.log(`Getting lat/lon and weather for city ${state.city}`);
+  getLatAndLon(state.city)
+    .then((latLon) => getWeatherKelvin(latLon.lat, latLon.lon))
+    .then((temp) => {
+      state.temperature = state.isF ? convertKtoF(temp) : convertKtoC(temp);
+      colorChange();
+    });
+};
+
+const changeSky = (event) => {
+  const inputSky = document.getElementById('skySelect');
+
+  const bg = document.getElementsByTagName('body')[0];
+
+  switch (inputSky.value) {
+    case 'Lover':
+      bg.style.background = 'url("assets/lover.jpeg")';
+      break;
+    case 'rep':
+      bg.style.background = 'url("assets/rep backup.jpeg")';
+      break;
+    case '1989':
+      bg.style.background = 'url("assets/1989 sky.jpeg")';
+      break;
+    case 'Red':
+      bg.style.background = 'url("assets/red.jpeg")';
+      break;
+    case 'Speak Now':
+      bg.style.background = 'url("assets/speak now.jpeg")';
+      break;
+    case 'fearless':
+      bg.style.background = 'url("assets/fearless.jpeg")';
+      break;
```

Comment:
Check out the logic on lines 94-111, see how similar the branches are of the switch statement. We look for some input value and then pick a background accordingly. What if we had a list of objects that we could iterate through to find the values. We could set up something like

```javascript
const SKY_BACKGROUND = [ 
  { input: 'Speak Now', background: 'url("assets/speak now.jpeg")'}, 
  { input: 'fearless', background: 'url("assets/fearless.jpeg")}
];
```

Then your method would iterate through `SKY_BACKGROUND` and find the first record that matches the input value, then use it as the source to pick which background to render, which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more backgrounds in the future, which would prevent you from having a really long switch statement